ui/pages/onboarding-flow/privacy-settings/privacy-settings.js

```diff
@@ -138,10 +164,16 @@ export default function PrivacySettings() {
     setUseTransactionSimulations(isTransactionSimulationsEnabled);
     dispatch(setPetnamesEnabled(turnOnPetnames));
 
+    // Profile Syncing Setup
     if (externalServicesOnboardingToggleState) {
-      if (!isProfileSyncingEnabled && participateInMetaMetrics) {
+      if (
+        !profileSyncingProps.isProfileSyncingEnabled &&
+        participateInMetaMetrics
+      ) {
         dispatch(performSignIn());
       }
+    } else {
+      profileSyncingProps.setIsProfileSyncingEnabled(false);
```

Comment:
Just ensures that we disable profile syncing if basic functionality is disabled.